client/packages/common/src/ui/layout/tables/components/Header/Header.tsx

```diff
@@ -141,6 +147,16 @@ export const HeaderCell = <T extends RecordWithId>({
     child
   );
 
+  const stickyStyles = isSticky
+    ? {
+        position: 'sticky' as const,
+        left: stickyPosition,
+        backgroundColor: 'white',
+        // When position is at 0, need to add some padding to avoid overlap
+        zIndex: `calc(${stickyZIndex} + 200)`,
```

Comment:
How does the padding apply to z index?

Reply:
Cause sum for position will be 0 if there's only 1 column since we want left = 0~ so just added some extra px to make sure the text in the header doesn't overlap

Reply:
Oh yeah true XD